runtime/src/main/java/org/corfudb/runtime/view/replication/ChainReplicationProtocol.java

```diff
@@ -166,7 +167,7 @@ protected void propagate(RuntimeLayout runtimeLayout,
                             .fillHole(token), OverwriteException.class);
                 }
             } catch (OverwriteException oe) {
-                log.trace("Propogate[{}]: Completed by other writer", globalAddress);
+                log.trace("Propagate[{}]: Completed by other writer", globalAddress);
```

Comment:
This is a rare event. I think we should make it INFO

Reply:
I agree with this, this should be at info level.